variants/deploy_with_capistrano/template.rb

```diff
@@ -0,0 +1,179 @@
+puts "=" * 80
+puts "Starting variant: deploy_with_capistrano"
+puts "=" * 80
+
+append_to_file("Gemfile") do
+  <<~'EO_RUBY'
+
+    # Deployment
+    #
+    # We tell bundler not to require these gems by default because the code that
+    # requires them will explicitly require them.
+    #
+    gem "capistrano", require: false
+    gem "capistrano-bundler", require: false
+    gem "capistrano-rails", require: false
+    gem "capistrano-rbenv", require: false
+    gem "capistrano-rake", require: false
+
+    # Capistrano fails when run from Github Action CI unless these gems are
```

Comment:
afaik it'll fail in general unless they're installed (I've always had it fail for me locally even on latest Ruby)

Reply:
hmmm maybe it depends on your key format? Anyway I should update the comment